bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala

```diff
@@ -58,8 +58,14 @@ sealed trait MagnitudesFilter {
   def ignoredMagnitudeField(v: FieldId): Boolean = false
   // Indicates if a magnitude is valid
   def validMagnitude(m: Magnitude): Boolean = !(m.value.isNaN || m.error.exists(_.isNaN))
-  // Find what band the field descriptor should represent
-  def findBand(id: FieldId, band: String): Option[MagnitudeBand] = MagnitudeBand.all.find(_.name == band)
+
+  // Find what band the field descriptor should represent, in general prefer "upper case" bands over "lower case" Sloan bands.
+  // This will prefer U, R and I over u', r' and i' but will map "g" and "z" to the Sloan bands g' and z'.
+  def findBand(id: FieldId, band: String): Option[MagnitudeBand] =
```

Comment:
Is it correct to prefer `R` over `r'`. In AGS we want it the other way around though this is at the level of VoTable, it would be great to confirm that a catalog won't expose both `R` and `r'`